tools/test-manylinux.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+#
+#
+# Tests the manylinux wheels on a plethora of bare-bone Linux docker images.
+
+set -ex
+
+echo "$0 running from $(pwd): $*"
+
+function setup_centos {
+    # CentOS container setup
+    yum install -q -y python epel-release
+    yum install -q -y python-pip
+}
+
+function setup_ubuntu {
+    # Ubuntu container setup
+    apt-get update
+    apt-get install -y python python-pip
+}
+
+
+function run_single_in_docker {
+    # Run single test inside docker container
+
+    # Detect OS
+    if grep -qi centos /etc/system-release /etc/redhat-release ; then
+        setup_centos
+    elif grep -qi ubuntu /etc/os-release ; then
+        setup_ubuntu
+    else
+        echo "WARNING: Don't know what platform I'm on: $(uname -a)"
+    fi
+
+    # Make sure pip itself is up to date
+    pip install -U pip
+    hash -r # let go of previous 'pip'
+
+    # Install modules
+    pip install confluent_kafka --no-index -f /io/wheelhouse
+    pip install pytest
+
+    # Verify that OpenSSL and zlib are properly linked
+    python -c '
+import confluent_kafka
+
+p = confluent_kafka.Producer({"ssl.cipher.suites":"DEFAULT",
+                              "compression.codec":"gzip"})
+'
+
+    pushd /io/tests
+    # Remove cached files from previous runs
+    rm -rf __pycache__ *.pyc
+    # Test
+    pytest --import-mode=append --ignore=avro
+    popd
+
+}
+
+function run_all_with_docker {
+    # Run tests in all listed docker containers.
+    # This is executed on the host.
+
+    [[ ! -z $DOCKER_IMAGES ]] || \
+        DOCKER_IMAGES="ubuntu:14.04 ubuntu:devel centos:6.6 centos:latest"
```

Comment:
probably all the ubuntu LTS versions? a centos 7 version that's explicitly 7 instead of relying on `latest` (which will eventually change to 8)? do we want to use pure debian as well since these are so easy to add/test here?

Reply:
Sure.

Turns out that centos 6.6 is stuck at Python 2.6, which we don't support (and pip is deprecating).